### PR TITLE
Repackage Protobuf definitions

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR performs repackaging of Protobuf definitions according to [new conventions](https://spine.io/docs/guides/naming-conventions.html).

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@dmdashenkov PTAL.

File: users/src/main/java/io/spine/users/role/RoleAggregate.java [hunk 1]
```diff
@@ -18,23 +18,27 @@
...
+import io.spine.users.role.event.RoleDeletedVBuilder;
+import io.spine.users.user.UserPart;
 
 /**
  * A role that can be assigned to {@linkplain UserPart users} and
  * {@linkplain GroupPart groups}, to perform access control.
  *
  * <p>Roles are assigned to {@link UserPart users} and {@link GroupPart groups} directly
- * and explicitly (please see {@link AssignRoleToUser} and {@link AssignRoleToGroup} commands).
+ * and explicitly (please see {@code AssignRoleToUser} and {@code AssignRoleToGroup} commands).
```
Comment (dmdashenkov) [inline users/src/main/java/io/spine/users/role/RoleAggregate.java]: Please keep `link`s. It's easier to rename messages that way.

File: users/src/main/java/io/spine/users/group/GroupPart.java [hunk 2]
```diff
@@ -18,21 +18,47 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.users.c.group;
+package io.spine.users.group;
```
Comment (dmdashenkov) [inline users/src/main/java/io/spine/users/group/GroupPart.java]: We've established that it'd be better if Java implementation was placed under a separate package `server`. This way, `GroupPart` should be placed under `io.spine.users.server.group`. This change does not affect the generated code.

**Comment by dmytro-grankin** [comment]
@dmdashenkov PTAL.

File: users/src/main/java/io/spine/users/package-info.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * This package contains the {@code Users} bounded context.
+ */
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+package io.spine.users;
```
Comment (dmdashenkov) [inline users/src/main/java/io/spine/users/package-info.java]: Please annotate this package with `@BoundedContext("Users")`. If there already are such annotations in subpackages, please remove those.

**Comment by dmytro-grankin** [comment]
@dmdashenkov PTAL.

**Pull request merged**